go/cmd/dolt/commands/version.go

```diff
@@ -203,17 +203,17 @@ func checkAndPrintVersionOutOfDateWarning(curVersion string, dEnv *env.DoltEnv)
 func getLatestDoltReleaseAndRecord(path string, dEnv *env.DoltEnv) (string, errhand.VerboseError) {
 	client := github.NewClient(nil)
 	release, resp, err := client.Repositories.GetLatestRelease(context.Background(), "dolthub", "dolt")
-	if err != nil || resp.StatusCode != 200 {
-		return "", errhand.BuildDError("error: failed to verify latest release").AddCause(err).Build()
-	}
-	releaseName := strings.TrimPrefix(*release.TagName, "v")
+	if err == nil && resp.StatusCode == 200 {
+		releaseName := strings.TrimPrefix(*release.TagName, "v")
 
-	err = dEnv.FS.WriteFile(path, []byte(releaseName), os.ModePerm)
-	if err != nil {
-		return "", errhand.BuildDError("error: failed to update version check file").AddCause(err).Build()
-	}
+		err = dEnv.FS.WriteFile(path, []byte(releaseName), os.ModePerm)
```

Comment:
Why not delete this too? It's kind of silly that we tell the user we can't create a file, then don't even tell them where that file would be so they can check permissions.

Silently fail!